spec/restful_resource/rails_validations_spec.rb

```diff
@@ -72,13 +72,13 @@
     end
   end
 
-  context '#patch without errors' do
```

Comment:
`patch` was a typo and duplicating the tests (introduced here https://github.com/carwow/restful_resource/pull/42)
The real `patch` tests are above